sys/include/net/hdlc/fcs.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2016 José Ignacio Alamos <jialamos@uc.cl>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    net_hdlc_fcs    FCS calculation
```

Comment:
@jia200x which is it then? According to all sources I can find HDLC uses CCITT (`0x1021`), too. Anyway, here's a more flexible (with regards to polynomials) and more lightweight implementation of CRC16: https://github.com/RIOT-OS/RIOT/pull/6112